packages/@okta/vuepress-site/docs/concepts/third-party-cookies/index.md

```diff
@@ -0,0 +1,47 @@
+
+When customer-hosted code makes a call to Okta that relies on an [Okta Session Cookie](/docs/guides/session-cookie/) being included in the HTTP request by the user’s browser, Safari blocks the cookie from reaching Okta because the user is on the customer domain and the cookie is being sent to Okta, a different domain.
+
+The specific functionality that is affected is session management, as well as token renewal in the OAuth 2.0 implicit flow.
+
+### Session management in customer-built applications
+
+If you host your own sign-in page, and use a self-hosted instance of the Okta Sign-In Widget to sign users in, or rely on JavaScript running in the user’s browser to make calls to Okta to handle session management, functionality can be broken by Safari’s blocking of third-party cookies.
+
+Okta session cookies that accompany XHR calls to Okta API endpoints like `/sessions/me` and `/user/me`	 are blocked by Safari because they are sent to a different domain than the one the user is on. Okta session cookies never get through to Okta, and Okta cannot complete the request.
+
+The result is that Okta returns 403 forbidden errors to the user or repeatedly directs them back to the sign-in page.
+
+Specifically, this problem affects [certain methods](https://github.com/okta/okta-auth-js#third-party-cookies) of the Okta Auth JavaScript SDK. The [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget#okta-sign-in-widget) uses the affected Okta Auth JavaScript SDK methods internally. Any customer-developed code that directly makes XHR calls to the Okta [Sessions API](/docs/reference/api/sessions/) is also affected. 
+
+### Token renewal in customer-built SPAs that use the OAuth 2.0 implicit flow
+
+If your code uses the OAuth 2.0 implicit flow to handle token renewal (which typically occurs in the context of a SPA that uses the [implicit flow](/docs/guides/implement-implicit/)), Safari can block Okta session cookies from being sent, so that token renewal is never successfully completed.
+
+The result is that ID tokens and access tokens expire without being renewed, and users are prompted to sign in more frequently, with the frequency determined by the token expiry time.
+
+## Workaround: Custom URL Domain
+
+You can implement a workaround by using the [Customize the Okta URL domain](/docs/guides/custom-url-domain/) feature.
+
+By making your Okta org effectively part of the same domain as your application server from a browser’s perspective, Custom URL Domain moves Okta session cookies to a first-party context. Calls to Okta become calls within the same site, and Safari third-party cookie blocking is no longer triggered.
```

Comment:
We should make it clear that you need the application domain and login domain to be the same.

Reply:
Added the following:

For example, if your original Okta org is `companyname.okta.com`, and your app server is `app.companyname.com`, you would use the custom URL domain feature to give your Okta org a new URL like `auth.companyname.com`. This puts your app and your Okta org within the same site. It is okay for them to have different subdomains (in this example, `your-app` and `auth`); it is the domain that needs to be the same (in the example, `companyname.com`), in order for them to be considered as being on the same site.

Reply:
looks good! please update the above sentence to remove "safari" and just say "third-party cookies blocking..."